hadoop-ozone/ozonefs/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneClientAdapterImpl.java

```diff
@@ -322,6 +330,38 @@ public void makeQualified(FileStatus status, URI uri, Path path,
     return new IteratorAdapter(bucket.listKeys(pathKey));
   }
 
+  @Override
+  public Map<String, List<OzoneAcl>> getAcls(String keyName)
+      throws IOException {
+    OzoneObj obj = OzoneObjInfo.Builder.newBuilder()
+        .setBucketName(bucket.getName()).setVolumeName(volume.getName())
+        .setKeyName(keyName).setResType(OzoneObj.ResourceType.KEY)
+        .setStoreType(OZONE).build();
+    List<OzoneAcl> userResult = new ArrayList<OzoneAcl>();
+    List<OzoneAcl> groupResult = new ArrayList<OzoneAcl>();
+
+    ozoneClient.getObjectStore().getAcl(obj).forEach(ozoneAcl -> {
+      if (ozoneAcl.getType().equals(ACLIdentityType.USER)) {
+        userResult.add(ozoneAcl);
+      } else if (ozoneAcl.getType().equals(ACLIdentityType.GROUP)) {
```

Comment:
Sorry for identifying this late, do we also need to check the other ACLIdentityType here like ACLIdentityType.WORLD etc.?